rest-tests/src/test/resources/filters/tw-sf-filter.json

```diff
@@ -0,0 +1,5 @@
+[{
+		"path": "text",
+		"op": "contains",
+		"value": "#backendTest"
+	}]
```

Comment:
Lets try to generate filter step from code. Either by backend call or directly from model?